news/2023/2023-02-17-community-contributors-elite-nominators-2022.md

```diff
@@ -0,0 +1,125 @@
+
+Throughout the year, he has been a solid all-rounder, standing out especially for his ample involvement in nomination resets. All this, plus his consistency as a Beatmap Nominator is something we think is well worthy of the Elite Nominator title!
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/SMOKELIND.jpg)](https://osu.ppy.sh/users/9327302)
+
+[SMOKELIND](https://osu.ppy.sh/users/9327302) made a massive splash in the past year with the huge amount of variety in their nominations across older and newer mappers alike. Willing to delve into such a large variety is often daunting to other Beatmap Nominators, but is business as usual for SMOKELIND, leaving a diverse catalogue of Ranked maps in his wake.
+
+There is no doubt in our mind that SMOKELIND has deserved their spot as an Elite Nominator for 2022, backed up in no small part by these outstanding contributions to the game in the mapping and modding community!
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/Mirash.jpg)](https://osu.ppy.sh/users/2841009)
+
+A long time Beatmap Nominator, [Mirash](https://osu.ppy.sh/users/2841009) has stood out this year due to his great contributions to the Ranked section through the nomination of various full-version and marathon songs and large spreads with diverse difficulties, all while maintaining a very low reset record. This is a tough task for any Beatmap Nominator to tackle, as it involves more work that is largely considered optional and ignored by many others in the group.
+
+Through commitment to his craft and a clear vision for the future of mapping, Mirash has helped populate the Ranked section with diversity and quality, all while being spotless compared to his peers, absolutely warranting his spot in the Elite Nominator line-up for this year.
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/moonpoint.jpg)](https://osu.ppy.sh/users/9558549)
+
+Continuing his excellent performance from 2021, [moonpoint](https://osu.ppy.sh/users/9558549) (formerly known as Apo11o) has proven himself time and time again as a prudent nominator. He's constantly pushed the Ranked section in innovative ways, promoting many map styles and song genres that would likely be ignored otherwise. This was especially notable in 2022, with Apo11o scoring above most other Beatmap Nominators in terms of variety and total beatmap set length, even going out of his way to spot issues in qualified maps and reset their nominations on many occasions.
+
+moonpoint's spotless tenure as a Beatmap Nominator cannot (and will not) go unnoticed, leaving his spot among the Elite an inevitability.
```

Comment:
first paragraph has `Apo11o`, but second starts with `moonpoint`, seems like it should be consistent